ddtrace/tracer/stats.go

```diff
@@ -278,13 +279,22 @@ func (sb *rawBucket) Export() statsBucket {
 	return csb
 }
 
+type trilean int32
+
+const (
+	trilean_not_set trilean = iota
+	trilean_true
+	trilean_false
```

Comment:
snake_case? 🙈

Reply:
@felixge You are right, I didn't notice it. The trilean concept blinded me. We'll merge to improve this.